### DISCUSSION TITLE
Request for Turkish Language Support

### DISCUSSION THREAD

**Discussion by atakanhr** [0xJacky/nginx-ui#636] [Ideas]
I hope this message finds you well. First, I would like to thank you for your outstanding work on the Nginx-UI project. It has been incredibly useful to me and many others.

I am writing to request the addition of Turkish language support to the project. As Nginx is widely used by Turkish-speaking users, providing a localized experience would significantly enhance the user experience for this community.

I would be happy to assist in the translation process once the framework for language support has been implemented. If this is something you would consider adding, please let me know how I can support this effort.

Thank you for considering this request, and I look forward to your response.

**Comment by 0xJacky** [comment]
Dear @atakanhr,

Thank you for your kind words and for your interest in contributing to the Nginx-UI project.

We are more than happy to welcome your contribution to translate Nginx-UI into Turkish! You can use our Weblate platform for online translations at the following link:
https://weblate.nginxui.com/projects/nginx-ui/frontend/tr/

For further instructions, please refer to this discussion on GitHub:
https://github.com/0xJacky/nginx-ui/discussions/161

We appreciate your support and look forward to your contribution. Should you have any questions or need further assistance, please feel free to reach out.

Best regards,
Jacky
